src/full-screen/components/runtime.tsx

```diff
@@ -85,18 +85,17 @@ export const Runtime: React.FC = () => {
 
   const iframeStyle = setScaling();
 
-  const params = queryString.parse(location.search);
-  const hash = queryString.parse(location.hash);
-  const sharedDoc = hash.shared ? "#shared="+hash.shared : "";
-  const preUrl = params?.wrappedInteractive + sharedDoc;
-  const subinteractiveUrl = preUrl && encodeURI(preUrl);
+  const wrappedInteractive = queryString.parse(location.search)?.wrappedInteractive;
 
-  if (!subinteractiveUrl) {
+  if (!wrappedInteractive) {
     return <div>No sub items available. Please add them using the authoring interface.</div>;
-  } else if (initMessage) {
+  } else {
+    const shared = queryString.parse(location.hash)?.shared;
+    const url = `${wrappedInteractive}${shared ? `#shared=${shared}` : ""}`;
```

Comment:
I think this should be removed. I know it was there before, but that seems like a mistake to me. 
The author should be escaping the URL correctly so there would be no need for the interactive to look for hash parameters on the full-screen interactive's URL.

Ie. the URL should have a search string like `?wrappedInteractive=https%3A%2F%2Fcodap.concord.org%23shared%3D1234`

Which would then get turned into `https://codap.concord.org#shared=1234`
`queryString.parse`

This way there is no special logic in the full-screen interactive, it is just passing through the wrappedInteractive parameter. 

No one is using this full-screen interactive so it is fine to change how it works.

Reply:
Resolved in https://github.com/concord-consortium/question-interactives/pull/156/commits/8bc09db1edfc1d38a7469952bfec980f817ffda1